tasks/cnm-to-cma/tests/test_cnm2cma.py

```diff
@@ -0,0 +1,103 @@
+from unittest.mock import patch
+from cnm_to_cma import CNM2CMA
+import json
+from typing import List
+
+
+class TestCNM2CMA:
+    def setup_method(self):
+        with patch("src.cnm2cma.cnm_to_cma.CNM2CMA.__init__", return_value=None):
+            self.cnm2cma = CNM2CMA()
+
+    def test_get_input_files(self):
+        product = {"files": [{"name": "file1"}, {"name": "file2"}]}
```

Comment:
Consider making the input more realistic to the schema and values we would expect to get.